yarn.lock

```diff
@@ -25172,11 +25145,16 @@ typescript-json-schema@^0.50.1:
     typescript "~4.2.3"
     yargs "^16.2.0"
 
-typescript@^4.0.3, typescript@~4.1.3, typescript@~4.2.3:
+typescript@^4.0.3, typescript@~4.2.3:
   version "4.2.4"
   resolved "https://registry.npmjs.org/typescript/-/typescript-4.2.4.tgz#8610b59747de028fda898a8aef0e103f156d0961"
   integrity sha512-V+evlYHZnQkaz8TRBuxTA92yZBPotr5H+WhQ7bD3hZUndx5tGOa1fuCgeSjxAzM1RiN5IzvadIXTVefuuwZCRg==
 
+typescript@~4.3.2:
```

Comment:
@Rugvip does this matter that we're now bringing in two versions of typescript?

Reply:
This one should end up being installed under the api extractor, so no. In the end I'm trusting the tests tho 😁